compiler.rst

```diff
@@ -419,7 +461,27 @@ Important Files
 
     asdl_c.py
         "Generate C code from an ASDL description."  Generates
-        :file:`Python/Python-ast.c` and :file:`Include/Python-ast.h`.
+        :file:`Python/Python-ast.c` and :file:`Include/Internal/pycore_ast.h`.
+
+    parser.c
+        The parser generated by :file:`Tools/peg_generator/pegen/c_generator.py`
+        from the grammar :file:`Grammar/python.gram`.  Creates the AST from
+        source code.  Rule functions for their corresponding production rules
+        are found here.
+
+    peg_api.c
+        Contains high-level functions to create an AST from source code which
+        are used by the interpreter.
```

Comment:
```suggestion
        Contains high-level functions, that are part of the C API, to create an AST from source code which
        are used by the interpreter.
```

Reply:
Hmm I think they were removed from the C API https://github.com/python/cpython/commit/57364ce34e0492fbc8b0a6b8c882f384bb489457.

Reply:
Oh, you're right!